_posts/2019-12-16-database-migration-in-self-contained-systems.md

```diff
@@ -75,51 +80,61 @@ data:
   spring.datasource.driver-class-name: org.postgresql.Driver
   spring.datasource.url: jdbc:postgresql://postgres:5432/post_service
 ```
-Secret
+
+A `Secret` is a Kubernetes Object for storing sensitive data. Similar to a `ConfigMap`, a `Secret` can also be read at runtime: 
+
 ```yaml
+# base/secret.yml
 apiVersion: v1
 kind: Secret
 metadata:
   name: post-secret
   namespace: migration
 type: Opaque
 data:
-  spring.datasource.password: bXlfc2VydmljZV9wYXNzd29yZA== #my_service_password
+  spring.datasource.password: bXlfc2VydmljZV9wYXNzd29yZA==
 ```    
 
 As we can see, the data from `post-configmap` and the password from `post-secret` are the data for connecting to the database
-with the user `post_service`. It means, after deploying the Spring Boot application, these data should be read and used by it.  
+with the user `post_service`. After deploying the Spring Boot application, our application and database should be connected and ready to use.
 
 ### Database
-With the scripts from `k8s/postgres` we create a PostgreSQL Database as
-[StatefulSet](https://cloud.google.com/kubernetes-engine/docs/concepts/statefulset) with an admin user. The password for
-the user is read from another secret. `StatefulSet` is a Kubernetes Object, that defines a system with unique, persistent identities 
-and with persistent disk storage. A database is exactly the case, where we should use a `StatefulSet`. 
+
+With the scripts from `k8s/postgres` we create a PostgreSQL Database as a
+[StatefulSet](https://cloud.google.com/kubernetes-engine/docs/concepts/statefulset) with an admin user.
+A `StatefulSet` is a Kubernetes Object that defines components with unique, persistent identities 
+and with persistent disk storage. A database fits well into the use case of a `StatefulSet`. 
```

Comment:
Does it make sense to add the `yaml` code for the `StatefulSet` to complete this example? I didn't add it in my version, because it is much code and it is not more than just the creation of a PostgreSQL database.